docs/Concepts/Withdrawals.md

```diff
@@ -0,0 +1,67 @@
+---
+description: Describe withdrawals
+---
+
+# Withdrawals
+
+Validators staking ether on Mainnet after [The Merge](Merge.md), accrue 2 forms of rewards:
+
+- Execution layer rewards paid directly to a withdrawal address (Ethereum address).
+- Consensus layer rewards for performing actions each epoch.
+
+The Capella upgrade implements an automated process that allows a validator's rewards to be transferred
+from the consensus layer to an Ethereum address on the execution layer. Before the Capella
+upgrade, consensus layer rewards were locked, and unable to be transferred to an Ethereum address.
+
+!!! important
+
+    When you create a validator, it's currently possible to set its withdrawal address to a
+    [BLS address](https://en.wikipedia.org/wiki/BLS_digital_signature), or an Ethereum address. After
+    the Capella fork, users must use an Ethereum address as their withdrawal address to allow automatics
```

Comment:
typo: `automatic`

Reply:
i think the same as below, maybe we just delete the sentence stating 'must use ethereum address', to the best of my knowledge can still setup with 0x00 credentials if you so desire...

Reply:
@rolfyone, can I update this to, "After the Capella upgrade, only Ethereum adresses will accrue rewards automatically."?

Reply:
ahm... i think it's worth being specific when we say 'accrue rewards', because to me that description would sound like you dont acrue rewards if you're using BLS...
Basically the accrual doesnt change, it's just whether we can take excess ETH and transfer it to an eth address, or whether it will just build on the validator balance over time and not get transferred out...